active/PipsPager/PipsPager.md

```diff
@@ -0,0 +1,349 @@
+
+#### Create a PipsPager
+
+A PipsPager with five visible pips that can be selected by the user to jump to that specific page.
+Additionally, the user can use navigational buttons; previous and next, to change the selected page incrementally. 
+Navigating beyond the first or last item does not wrap around.
+By default, the navigational buttons are collapsed, the pips are oriented horizontally,
+and the total number of pages is infinite.
+
+![A prototype of five horizontal dots with the first selected.](images/PipsPager_Default.png)
+
+XAML
+```XAML 
+<muxc:PipsPager x:Name="DefaultPipsPager" />
+```
+
+#### PipsPager with visible navigational buttons
+
+The visibility of the previous and next buttons can be set via the
+`PreviousButtonVisibility` and `NextButtonVisibility` properties. 
+These buttons allow the user to incrementally select a page. The possible values are:
+- Visible: The button is visible and enabled, except it is hidden at the extents.
+  - For example, when the current page is the first page, the previous button is hidden.
+  - Note: Hidden means that the button is invisible but does take up layout space.
+- VisibleOnPointerOver: The behavior is the same as Visible *except* that the button is only Visible when
+the user is hovering over the paging UI with their cursor.
+- Collapsed: The button is not visible to the user and does **not** take up layout space.
+
+![A sketch of five horizontal dots with the first selected and the previous and next buttons are visible](images/PipsPager_Visible.gif)
+
+XAML
+```XAML 
+<muxc:PipsPager x:Name="VisibleButtonPipsPager"
+  NumberOfPages="5"
+  PreviousButtonVisibility="Visible"
+  NextButtonVisibility="Visible" />
+```
+
+#### Vertical PipsPager with VisibleOnPointerOver button visibility
+
+The PipsPager can also be oriented vertically which does not change its behavior or interaction.
+The various button visibility modes can apply to either orientation.
+The below example highlights the VisibleOnPointerOver button visibility.
+
+![A prototype of five vertical dots with the second initially selected and the previous and next buttons appearing and disappearing based on user hover and the selected page](images/PipsPager_VisibleOnPointerOver.gif)
+
+XAML
+```XAML 
+<muxc:PipsPager x:Name="VerticalPipsPager"
+  NumberOfPages="5"
+  Orientation="Vertical" 
+  PreviousButtonVisibility="VisibleOnPointerOver" 
+  NextButtonVisibility="VisibleOnPointerOver" />
+```
+
+#### Number of pages displayed less than total
+
+If the number of pages of content is large and do not need to be navigated to at once,
+the MaxVisiblePips property can be set to limit the number of visible, interactable, pips.
+If the NumberOfPages is greater than the MaxVisiblePips then the pips will scroll so that the selected page is
+centered in the control. If the NumberOfPages is equal to or less than the MaxVisiblePips,
+no scrolling will occur and the number of pips shown will be the same as the NumberOfPages.
```

Comment:
If NumberOfPages is less than MaxVisiblePips, is space reserved for the unused pips? Or does it collapse out?

If space is reserved, does that extra space go on the right? Or do the pips center?

E.g. NumberOfPages = 3, MaxVisiblePips = 5

Option: Pips are centered in reserved space, arrows stay put.
| ◀ &nbsp; • • • &nbsp; ▶ |

Option: Pips are left-aligned in reserved space, arrows stay put.
| ◀ • • • &nbsp; &nbsp; ▶ | 

Option: Space collapses out.
| ◀ • • • ▶ |

Option: Pips are spread evenly in reserved space.
| ◀ • &nbsp; • &nbsp; • ▶ |

Option: Pips and arrows center in reserved space.
| &nbsp; ◀ • • • ▶ &nbsp; |

Option: Pips and arrows left-aligned in reserved space.
| ◀ • • • ▶ &nbsp; &nbsp; |

Reply:
Discussion: The layout space is the minimum of Number and Max.

Recommend: add a comment to the description.